site/src/docs/patterns/forms/form-validation.mdx

```diff
@@ -256,12 +336,88 @@ In some cases, multiple input fields can be related to each other in a way that
 
 An example of this could be having two date inputs for setting a start and an end time. If the end time is set to be earlier than the start time, both date inputs become erroneous. Since assistive technologies only see one input at a time, all related inputs should be set to an error state and given an appropriate error message. The error message can be the same for all inputs or more input specific if the situation allows so. All error messages are also added to the error summary list. This method has been illustrated in the image below.
 
-<Image
-  src="/images/patterns/form/validation/multiple-fields-validation@2x.png"
-  alt="Showing error messages of multiple input fields"
-  style={{ maxWidth: 482 }}
-  viewable
-/>
+<Playground scope={{ Button, ErrorSummary, DateInput }} >
+
+```jsx
+import { Button, ErrorSummary, DateInput } from 'hds-react';
+
+() => {
+  const [formValues, setFormValues] = React.useState({});
+  const [formErrors, setFormErrors] = React.useState({});
+  const [isSubmitted, setIsSubmitted] = React.useState(false);
+  const errorList = Object.entries(formErrors);
+
+  const isSameOrAfter = (dateA, dateB) => dateA && dateB && dateA.valueOf() >= dateB.valueOf();
+
```

Comment:
For some reason these are accepted as valid inputs.

![image](https://github.com/City-of-Helsinki/helsinki-design-system/assets/2639230/e3c95ca7-ae56-42b1-915a-0e078d8b771e)

If end date has two digits, like 10.8., then error is shown.